### ISSUE TITLE
Bridges starts in free air

### ISSUE THREAD

**Issue by simonbuehler** [SoftFever/OrcaSlicer#1340]
**Describe the bug**
When printing a overhang the bridge starts disconnected in the air, already switched to infill/inner/outer

**3mf File for This Bug**
[sprocketwheel-Body.zip](https://github.com/SoftFever/OrcaSlicer/files/11743400/sprocketwheel-Body.zip)

**To Reproduce**
slice the file

**Expected behavior**
the bridge should start connected (there is enough space)

**Screenshots**
![grafik](https://github.com/SoftFever/OrcaSlicer/assets/78061/e2171942-3fc6-421b-ba97-bfc302c9e5e3)

**Printer model**
p1p
**Desktop (please complete the following information):**
 - OS: win10
 - Version 1.6.3

**Comment by tastyratz** [comment]
![image](https://github.com/SoftFever/OrcaSlicer/assets/657895/9128c154-d050-4b2c-b312-caf188f5eecf)

I've also been experiencing this and it drives me nuts. A perfect example is this Nevermore XL cartridge.

Orcaslicer is trying to free air print bridges and connect them second. This obviously does not work and the result is a ton of dropped bridges and the mesh looking like garbage.

Changing wall order does not make any difference.

This is still happening in 1.7.0

**Comment by tastyratz** [comment]
Also to note that this behavior at least in my old copy of superslicer is not happening.

![image](https://github.com/SoftFever/OrcaSlicer/assets/657895/40724a44-5d0f-4dae-8918-90581757ba53)

**Comment by SoftFever** [comment]
> Also to note that this behavior at least in my old copy of superslicer is not happening consistently, but, does seem to happen on the first bridge of each row.
> 
> This is my example file: https://github.com/nevermore3d/Nevermore_Micro/blob/master/V6/STL/XL_HEPA_Cartridge/1_XL_Cartridge.stl
> 
> ![image](https://user-images.githubusercontent.com/657895/282171555-40724a44-5d0f-4dae-8918-90581757ba53.png)
> 
> The path on superslicer is making a row of rectangles and incorporating the outside wall to the bridge path. Orcaslicer is performing all bridging operations first and THEN creating all outside walls.
> 
> The order of operations in Orcaslicer is causing bridge failures. Let me know if I could be better explaining this one @SoftFever
> 
> I tried this with classic with and without detect thin walls and arachne, same patterning.

@tastyratz 
thanks for the information.
Yeah, I checked with PrusaSlicer as well, it also has this issue.
Need more investigation for such edge cases.

**Comment by tastyratz** [comment]
I was really hoping https://github.com/SoftFever/OrcaSlicer/pull/3319 would also resolve this, but, unfortunately not the case as verified with 2.0 trying to slice the nevermore cartridge just now. I tried disabled, limited, and none but it still bridges over open air. 
![image](https://github.com/SoftFever/OrcaSlicer/assets/657895/70432100-1a5e-4197-b7e4-cab7cde6b6a5)

I did, however, just figure out that the problem ONLY happens with Arachne and does NOT happen when using classic wall generator.

**Comment by netweaver1970** [comment]
for me, on 2.0 I  get the same behaviour. I'm also on arachne.  It happens on higher-up layers as well, as long as they are single line bridges, they are all floating. Actually they are identified as overhang wall line type, maybe that's the issue? 

I didn't check classic yet. Maybe something to do with these very thin (single line) walls? 
Double line walls, on top of the single line bridges below, are properly handled with a serpentine style of extrusion.

Anyway, easy enough to replicate. So it can be debugged, by people smarter than me :)

![image](https://github.com/SoftFever/OrcaSlicer/assets/18028618/8bcfd5c5-2113-4896-a59e-d8d323d20623)

![image](https://github.com/SoftFever/OrcaSlicer/assets/18028618/f8a44b62-4bd4-4959-b778-bcd467e29f16)

![image](https://github.com/SoftFever/OrcaSlicer/assets/18028618/df97a26e-c3f4-4224-947c-dd9487894a6c)

Higher up:
![image](https://github.com/SoftFever/OrcaSlicer/assets/18028618/fe8521c6-217d-477e-9850-a7527fe7d225)

**Comment by tastyratz** [comment]
I have done some additional GIT digging and found this https://github.com/supermerill/SuperSlicer/issues/3567

I've replicated this superslicer bug myself on Orca 2.0 by changing the perimeter count (wall loops) from 3 to 1. Once I dropped to 1 perimeter then Arachne is generating the bridges AFTER drawing the perimeter with the little nipples to attach to. This appears to be the correct behavior to me on layer 299.

![image](https://github.com/SoftFever/OrcaSlicer/assets/657895/fc92765d-f6c2-49f6-9388-af92cc83ee97)

I'm hoping by linking these reports together and presenting my findings across projects that someone on this or other bug reports connects the dots.

This seems to be further narrowed down to Arachne perimeter order when single wall bridges are present and more than 1 perimeter/wall loop is selected.

I also did MORE investigation and changed Wall printing order to Outer/Inner.

When wall printing order is Inner/Outer, the bridges fail in open air. 
If wall printing order is Outer/Inner, it prints the supporting bridge wall first on layer 299 but only sorta fixes things on layer 314 which behaves extremely strange
![image](https://github.com/SoftFever/OrcaSlicer/assets/657895/0a75e307-7c7c-48aa-b18e-d9fe2b458d39)
Some bridges then are drawn before perimeters/open air, some are drawn after perimeters.


Ultimately: 

This could be solved by forcing "Print bridges after perimeters" either by an option or default behavior as I can't see why you might ever want to print bridges before perimeters.

If we can detect bridges, this seems like a viable solution to present.
